docs/src/extend/plugins.md

```diff
@@ -191,9 +260,16 @@ ESLint plugins should be linted too! It's suggested to lint your plugin with the
 
 ## Share Plugins
 
-In order to make your plugin available to the community you have to publish it on npm.
+In order to make your plugin available publicly, you have to publish it on npm. When doing so, please be sure to:
 
-To make it easy for others to find your plugin, add these [keywords](https://docs.npmjs.com/cli/v9/configuring-npm/package-json#keywords) to your `package.json` file:
+1. **List ESLint as a peer dependency.** Because plugins are intended for use with ESLint, it's important to add the `eslint` package as a peer dependency. To do so, manually edit your `package.json` file to include a `peerDependencies` block, like this:
+
+    ```json
+    {
+        "peerDependencies": {
+            "eslint": ">=8.0.0"
+        }
+    }
+    ```
 
-* `eslint`
-* `eslintplugin`
+2. **Specify keywords.** ESLint plugins should specify `eslint` and `eslint-plugin` as  [keywords](https://docs.npmjs.com/cli/v9/configuring-npm/package-json#keywords) in your `package.json` file.
```

Comment:
```suggestion
2. **Specify keywords.** ESLint plugins should specify `eslint`, `eslintplugin` and `eslint-plugin` as [keywords](https://docs.npmjs.com/cli/v9/configuring-npm/package-json#keywords) in your `package.json` file.
```

In the tutorial, we recommend `eslintplugin` as well. Most plugins indeed have all three keywords.

https://www.npmjs.com/search?q=keywords:eslintplugin